repository-data/application/src/main/resources/hcm-config/namespaces/hee-cms-platform.cnd

```diff
@@ -2,5 +2,14 @@
 <'hippo'='http://www.onehippo.org/jcr/hippo/nt/2.0.4'>
 <'hippostd'='http://www.onehippo.org/jcr/hippostd/nt/2.0'>
 <'hippostdpubwf'='http://www.onehippo.org/jcr/hippostdpubwf/nt/1.0'>
+<'hippotranslation'='http://www.onehippo.org/jcr/hippotranslation/nt/1.0'>
+
+[hee:basedocument] > hippo:document, hippostd:publishableSummary, hippostdpubwf:document
+  orderable
+
+[hee:guidance] > hee:basedocument, hippostd:relaxed, hippotranslation:translated
+  orderable
+
+[hee:link] > hee:basedocument, hippostd:relaxed, hippotranslation:translated
+  orderable
```

Comment:
@smanivasag Should we create the Link as document or compound type ? I think the link document will rarely be reused. So if we keep it as compound type and and embed it into the main document may make our content structure neater.

Reply:
Yes, go ahead. I guess we can wrap it as a DocumentType in the future if required.

Would suggest to apply these changes on top of Adriana's latest branch feature/HEE-72-content-blocks